CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## v1.0.2
+### 2024-2-26
+
+This version of HOSS correctly handles edge-aligned geographic collections by 
+adding the attribute `cell_alignment` with the value `edge` to `hoss_config.json`
+for edge-aligned collections (namely, ATL16), and by adding functions that 
+create pseudo bounds for edge-aligned collections to make HOSS use the 
+`dimension_utilities.py` function, `get_dimension_indices_from_bounds`.
+
+This change also includes an addition of a CF override that addresses an 
+issue with the ATL16 metadata for the variables `/spolar_asr_obs_grid` and 
+`/spolar_lorate_blowing_snow_freq` where their `grid_mapping` attribute points
+to north polar variables instead of south polar variables. This CF Override 
+will have to be removed if/when the metadata is corrected. 
```

Comment:
Nit: It's a bit of a strong statement to say it will *have* to be removed. It can be (and I think it would be good to do so), but it's unlikely that it would break anything by having an override that has the same information as the attribute it's overriding.